languages/python/sqlalchemy-oso/sqlalchemy_oso/__init__.py

```diff
@@ -1,6 +1,7 @@
 __version__ = "0.0.1"
 
 from .auth import register_models
+from .oso import SQLAlchemyOso
 from .session import authorized_sessionmaker, set_get_session
 
 __all__ = ["register_models", "authorized_sessionmaker", "set_get_session"]
```

Comment:
Can we move `set_get_session` into the old roles module? Fine with that as a breaking change on both `main` and this branch.

Reply:
for sure